hs_core/tasks.py

```diff
@@ -39,11 +39,15 @@
 logger = logging.getLogger('django')
 
 
+# TODO: Currently there are two different cleanups scheduled.
+# TODO: One is 20 minutes after creation, the other is nightly.

```

Comment:
Are you suggesting we should only have one or the other with the TODOs?  If so, I think the nightly is sufficient and we can get rid of the 20 minute cleanup...

Reply:
That's why I put them all in one place locally.

Reply:
@sblack-usu However, putting things into a local directory doesn't allow even user resources to be zipped properly via current code, because said code **assumes** that the zip target is federated the same way as the zip source.